packages/css/src/card/card.scss

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-card {
+  display: grid;
+  gap: 0.5rem;
+  outline-offset: var(--amsterdam-card-outline-offset);
+  position: relative;
+  touch-action: manipulation;
+
+  /* Set native outline on card if card has focus visible within */
+  &:has(:focus-visible) {
+    outline-color: -webkit-focus-ring-color;
+    outline-style: auto;
+    outline-width: 1px;
+  }
+
+  /*
+    Remove link focus outline only if :has is supported,
+    which means the outline on the whole card is set
+  */
+  &:has(:focus-visible) a:focus {
+    outline: none;
+  }
+
+  /*
```

Comment:
Clever! I do wonder whether it’s going to hold up with more advanced cards, but let’s see how far we get.

Reply:
Yeah, I think we should be fine as long as we don't wrap the link in an element that has `position: relative`